shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -297,12 +299,12 @@ public boolean handleKeyEvent(KeyEvent event) {
         final int selStart = Selection.getSelectionStart(mEditable);
         final int selEnd = Selection.getSelectionEnd(mEditable);
         final int character = event.getUnicodeChar();
-        if (selStart < 0 || selEnd < 0 || character == 0) {
```

Comment:
is this case valid now, or just not possible because of the KEYCODE_DEL case above?